apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.Data/SpannerCommand.cs

```diff
@@ -42,7 +43,7 @@ public sealed class SpannerCommand : DbCommand
         public SpannerCommand()
         {
             DesignTimeVisible = true;
-            _commandTimeout = ConnectionPoolOptions.Instance.TimeoutMilliseconds;
+            _commandTimeout = (int)ConnectionPoolOptions.Instance.Timeout.TotalSeconds;
```

Comment:
yes that was a bug that I caught that *ahem* wouldnt have happened if I used timespan originally.